ddtrace/contrib/aws_lambda/patch.py

```diff
@@ -199,5 +199,5 @@ def unpatch():
     except AttributeError:
         return
     except Exception:
-        log.exception("Error unpatching AWS Lambda")
+        log.debug("Error patching AWS Lambda", exc_info=True)
```

Comment:
Can we add more details to this debug log. It's not clear what the impact is. Is the impact no longer tracing lambdas that timeout?

```suggestion
        log.debug("Error patching AWS Lambda")
```